pkg/proxy/handlers/spacelister_list.go

```diff
@@ -6,20 +6,24 @@ import (
 	"net/http"
 	"time"
 
-	toolchainv1alpha1 "github.com/codeready-toolchain/api/api/v1alpha1"
-	"github.com/codeready-toolchain/registration-service/pkg/context"
-	"github.com/codeready-toolchain/registration-service/pkg/proxy/metrics"
 	"github.com/labstack/echo/v4"
 	errs "github.com/pkg/errors"
 	apierrors "k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/apimachinery/pkg/selection"
+
+	toolchainv1alpha1 "github.com/codeready-toolchain/api/api/v1alpha1"
+	"github.com/codeready-toolchain/registration-service/pkg/context"
+	"github.com/codeready-toolchain/registration-service/pkg/proxy/metrics"
+	"github.com/codeready-toolchain/registration-service/pkg/signup"
 )
 
 func HandleSpaceListRequest(spaceLister *SpaceLister) echo.HandlerFunc {
 	return func(ctx echo.Context) error {
 		// list all user workspaces
 		requestReceivedTime := ctx.Get(context.RequestReceivedTime).(time.Time)
+		ctx.Set(context.PublicViewerEnabled, false) // disable public-viewer on list endpoint
```

Comment:
Are you just temporally disabling it until other pieces are in?

Reply:
no, this will always be disabled by default as we don't want to return the list of all community visible workspaces with this endpoint; just the ones the user has direct access to.

Reply:
OK.